### PR TITLE
Update upgrading-from-400-to-410.md

### PR DESCRIPTION
**Pull request by samgnaniah**
Fixes https://github.com/wso2/docs-apim/issues/6221

### CODE REVIEW

**Comment by HiranyaKavishani** [comment]
Hi @samgnaniah,

There is missing note in the PR[1] 

[1] https://github.com/wso2/docs-apim/issues/6221#issuecomment-1244954125

Thanks!
Hiranya

**Comment by samgnaniah** [comment]
Updated with https://github.com/wso2/docs-apim/pull/6226/commits/f8265aaf73c85129819d0328b5dadf4063bb331f

File: en/docs/install-and-setup/upgrading-wso2-api-manager/400-to-410/upgrading-from-400-to-410.md [hunk 1]
```diff
@@ -150,6 +150,10 @@ Follow the instructions below to move all the existing API Manager configuration
 
 5. If you have enabled any other feature related configurations at `<API-M_4.0.0_HOME>/repository/conf/deployment.toml`, make sure to add them in to `<API-M_4.1.0_HOME>/repository/conf/deployment.toml` file.
 
+!!! note
+    If the older API-M setup has been configured for different admin roles other than admin and if the role is not persisted in read-only userstore, make sure not to change the `admin_role` configuration in the deployment.toml this time. You have to configure it after step 4.
```
Comment (HiranyaKavishani) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/400-to-410/upgrading-from-400-to-410.md:154]: different admin role

Comment (samgnaniah) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/400-to-410/upgrading-from-400-to-410.md]: Fixed with https://github.com/wso2/docs-apim/pull/6226/commits/cc17de584b449be8a21bdc48019fe20a538e772f

**Review by ruthryi — APPROVED** [review]

**Pull request merged**